From 0c47648529d6f57ff6562e2789111ee15acfb8a5 Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Mon, 13 Jan 2014 08:10:54 +0000
Subject: [PATCH 082/351] ASoC: wm_adsp: Make missing length info message
 sound less serious

Missing length info is not a serious issue so make the message sound a
little less ominous.

Change-Id: I1f1d92122466e3fe9f52b7720722c77277e3ec01
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Signed-off-by: Nikesh Oswal <Nikesh.Oswal@wolfsonmicro.com>
---
 sound/soc/codecs/wm_adsp.c |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/sound/soc/codecs/wm_adsp.c b/sound/soc/codecs/wm_adsp.c
index b941237..1947d9e 100644
--- a/sound/soc/codecs/wm_adsp.c
+++ b/sound/soc/codecs/wm_adsp.c
@@ -1465,7 +1465,7 @@ static int wm_adsp_setup_algs(struct wm_adsp *dsp)
 					region->len *= 4;
 					wm_adsp_create_grouped_control(dsp, region, true);
 				} else {
-					adsp_warn(dsp, "Missing length info for region DM with ID %x\n",
+					adsp_warn(dsp, "Length info not specified for region DM with ID %x\n",
 						  be32_to_cpu(adsp1_alg[i].alg.id));
 				}
 
@@ -1483,7 +1483,7 @@ static int wm_adsp_setup_algs(struct wm_adsp *dsp)
 					region->len *= 4;
 					wm_adsp_create_grouped_control(dsp, region, true);
 				} else {
-					adsp_warn(dsp, "Missing length info for region ZM with ID %x\n",
+					adsp_warn(dsp, "Length info not specified for region ZM with ID %x\n",
 						  be32_to_cpu(adsp1_alg[i].alg.id));
 				}
 			}
@@ -1521,7 +1521,7 @@ static int wm_adsp_setup_algs(struct wm_adsp *dsp)
 					region->len *= 4;
 					wm_adsp_create_grouped_control(dsp, region, true);
 				} else {
-					adsp_warn(dsp, "Missing length info for region XM with ID %x\n",
+					adsp_warn(dsp, "Length info not specified for region XM with ID %x\n",
 						  be32_to_cpu(adsp2_alg[i].alg.id));
 				}
 
@@ -1539,7 +1539,7 @@ static int wm_adsp_setup_algs(struct wm_adsp *dsp)
 					region->len *= 4;
 					wm_adsp_create_grouped_control(dsp, region, true);
 				} else {
-					adsp_warn(dsp, "Missing length info for region YM with ID %x\n",
+					adsp_warn(dsp, "Length info not specified for region YM with ID %x\n",
 						  be32_to_cpu(adsp2_alg[i].alg.id));
 				}
 
@@ -1557,7 +1557,7 @@ static int wm_adsp_setup_algs(struct wm_adsp *dsp)
 					region->len *= 4;
 					wm_adsp_create_grouped_control(dsp, region, true);
 				} else {
-					adsp_warn(dsp, "Missing length info for region ZM with ID %x\n",
+					adsp_warn(dsp, "Length info not specified for region ZM with ID %x\n",
 						  be32_to_cpu(adsp2_alg[i].alg.id));
 				}
 			}
-- 
1.7.9.5

